cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1294,7 +1294,7 @@
 
   <property>
     <name>messaging.system.topics</name>
-    <value>${audit.topic},${notification.topic}</value>
+    <value>${audit.topic},${metrics.messaging.topic.prefix},${notification.topic}</value>
```

Comment:
I'm using it as the actual topic for publish/fetch. Maybe I should remove "prefix"? I was trying to follow the same naming convention as `metrics.kafka.topic.prefix`